resources/views/user/home.blade.php

```diff
@@ -9,20 +9,59 @@
         <h2 class="d-flex justify-content-center">Find Creativity In</h2>
         <h3 class="d-flex justify-content-center">Gray World</h3>
     </div>
+
     <div class="d-flex justify-content-around trans-btn">
-        <form id="check-in" method="POST" action="{{ route('check-ins.store') }}">
-            @csrf
-            <button type="submit" class="btn btn-secondary">Check-in</button>
-        </form>
-        <form>
-            @csrf
-            <button type="submit" class="btn btn-secondary">Absent</button>
-        </form>
-        <form id="check-out" method="POST" action="{{ route('check-outs.store') }}">
-            @csrf
-            <button type="submit" class="btn btn-secondary">Check-out</button>
-        </form>
+        <button id="btn-check-in" class="btn btn-secondary" data-toggle="modal" data-target="#myModal">Check-in</button>
+        <button id="btn-absent" class="btn btn-secondary" data-toggle="modal" data-target="#myModal">Absent</button>
+        <button id="btn-check-out" class="btn btn-secondary" data-toggle="modal" data-target="#myModal">Check-out</button>
+    </div>
+
+    <!-- The Modal -->
+    @if (Auth::check())
+    <div class="modal" id="myModal">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <!-- Modal Header -->
+                <div class="modal-header">
+                    <h4 class="modal-title">Are you sure?</h4>
+                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                </div>
+                <!-- Modal body -->
+                <div class="modal-body">
+                    Confirm Before
+                </div>
+                <!-- Modal footer -->
+                <div class="modal-footer d-flex justify-content-around">
+                    <form id="form" method="POST" action="">
```

Comment:
you can leave out the form here, instead, set data on button element

```
<button id="confirm" type="submit" class="btn btn-danger" data-form="">Confirm</button>
```

after that, write js like
register event handler for button in modal
```
$('#confirm').click(function(e) {
    const formSelector = this.data('form')
    $(formSelector).submit()
});
```
when user click on button `#btn-check-in` --> set `data-form="#check-in"`

and you can handle the form submission from modal